numpy/core/_add_newdocs.py

```diff
@@ -3939,17 +3939,22 @@
 
     Constructs Python bytes showing a copy of the raw contents of
     data memory. The bytes object can be produced in either 'C' or 'Fortran',
-    or 'Any' order (the default is 'C'-order). 'Any' order means C-order
+    or 'K' or 'A' order (the default is 'C'-order). 'A' order means C-order
     unless the F_CONTIGUOUS flag in the array is set, in which case it
-    means 'Fortran' order.
+    means 'Fortran' order. 'K' order is as close to the order array elements appear
+    in memory.
 
     .. versionadded:: 1.9.0
 
     Parameters
     ----------
     order : {'C', 'F', None}, optional
         Order of the data for multidimensional arrays:
-        C, Fortran, or the same as for the original array.
+        'C' means it should be C contiguous,
+        'F' means it should be Fortran contiguous,
+        'A' means it should be 'F' if the inputs are all 'F', 'C' otherwise.
+        'K' means it should be as close to the layout of the inputs as
+        is possible, including arbitrarily permuted axes.
```

Comment:
This is better, but for consistency's sake I think it'd best if it matches the parameter description for `order` from `copy` exactly

Reply:
Got it. also, the description of parameters in the commit was taken from docstring of `nditer`. `copy`'s docstring is shorter. Which one should I use?

Reply:
Do you mean `ndarray.copy` instead of `nditer.copy`? The parameter description from either `ndarray.copy` (minus the part about how it differs from `np.copy`) or `np.copy` is fine - though the wording is slightly different, the underlying point is the same.

Reply:
Ok thanks

Reply:
Also, noticed that `copy` docstring doesn't have `returns` documentation. Should a bug be filed for it?

Reply:
You mean in `ndarray.copy`? Many of the docstrings for the `ndarray` methods simply refer to the corresponding function in the `numpy` namespace. `copy` generally follows that pattern and there is no need to add a `Returns` section to the method docstring.

Reply:
Yes `ndarray.copy`. Okayy, got it